tests/genmc/pass/litmus/2_2w_3sc_rel1.rs

```diff
@@ -15,19 +14,18 @@ use crate::genmc::*;
 
 #[unsafe(no_mangle)]
 fn miri_start(_argc: isize, _argv: *const *const u8) -> isize {
-    let _ids = unsafe { create_pthreads_no_params([thread_1, thread_2]) };
+    let _t0 = unsafe {
```

Comment:
You can simplify this further by using a single big unsafe block, and removing the unused `let` bindings.